front-end/src/components/SearchResult.vue

```diff
@@ -187,7 +193,7 @@ const downloadLabel = computed(
           <svg
             class="usa-icon"
             aria-hidden="true"
-            aria-labelledby="warning-icon"
+            aria-label="warning-icon"
```

Comment:
This should be "This contractor has indicated in SAM.gov that it provides or uses prohibited equipment. Please follow your agency’s procedures."

That is the value that previously was being referenced by the aria-labelledby with the ID of "warning-icon"